crates/fluvio-sc/src/config/sc_config.rs

```diff
@@ -29,6 +29,7 @@ pub struct ScConfig {
     pub namespace: String,
     pub x509_auth_scopes: Option<PathBuf>,
     pub white_list: HashSet<String>,
+    pub tls: bool,
```

Comment:
x509 requires TLS, so it redundant configuration

Reply:
Do you mean x509_auth_scopes? I can run a cluster with TLS without scopes.

Reply:
No I meant, why do need another `tis` config as TLS is already implied in the existing config.

Reply:
where?